sciencer/core.py

```diff
@@ -118,6 +225,9 @@ def iterate(
         Returns:
             List[Paper]: the resulting collection of papers
         """
+        callbacks = [] if callbacks is None else callbacks
+        callbacks.append(LogCallbacks())
```

Comment:
This addition should be described in the Pull Request. "New default logger when no callbacks are provided"